__tests__/models.test.js

```diff
@@ -174,3 +174,66 @@ describe('GET/api/articles/:article_id/comments', () => {
     })
   })
 })
+
+describe.only('POST/api/articles/:article_id/comments', () => {
```

Comment:
Please only push your code when you've removed any .only's and you know all your tests are passing